composer.json

```diff
@@ -34,7 +34,8 @@
         "symfony/console":  "~3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~5.0"
+        "phpunit/phpunit":    "~5.0",
+        "mikey179/vfsStream": "^1.6"
```

Comment:
This library reads from disk pretty much. I don't think it's overkill.

I also like to have few dependencies (wo doesn't?), but I don't think it's a huge deal since it's just a dev dependency.